scan.c

```diff
@@ -41,6 +45,26 @@ const char * iio_context_info_get_uri(
 	return info->uri;
 }
 
+#ifdef WITH_MATLAB_BINDINGS_API
```

Comment:
Could you put those two functions at the bottom of the file? So that they can be wrapped together with iio_create_scan_block() in the same #ifdef WITH_MATLAB_BINDINGS_API.